src/webgpu/api/operation/render_pipeline/pipeline_output_targets.spec.ts

```diff
@@ -139,7 +139,8 @@ g.test('color,attachments')
     const info = kTextureFormatInfo[format];
     await t.selectDeviceOrSkipTestCase(info.feature);
 
-    const writeValue = { R: 0.125, G: 0.25, B: 0.5, A: 1 };
+    const writeValue =
+      info.sampleType === 'float' ? { R: 0.2, G: 0.6, B: 0.8, A: 1 } : { R: 2, G: 4, B: 8, A: 16 };
```

Comment:
I meant a for each different attachment index, not for each component (although that is also good). I don't know how easy that would be to do though.

Reply:
Was making an attempt to use the new `bitsToNumber`/`numberToBits` conversion to take care of float/unorm precision issue for different value but doesn't fully work for 16bit float. https://github.com/gpuweb/cts/commit/cbbc37893d5c123b8f31043a785bf81e29f20de0#diff-1bb578801feab213197f6db552e57d91911c1e537576b8c3e3179d09ec59c5ccR182

Seems a good place for some more texture expect helper (within a threshold). Will try on later

Reply:
The texture helpers I wrote recently have fancy thresholding options, and we want to transition the old texture helper functions onto that ideally. #1121